tests/bench/beam_baseline.py

```
@@ -0,0 +1,558 @@
+load_dotenv(bench_dir / ".env")
+
+# OpenRouter model format for baseline testing
+MODEL_BEING_TESTED = "anthropic/claude-haiku-4.5"
+
+
+class BEAMBaselineRunner:
+    """
+    Executes BEAM benchmark tests directly against Claude Sonnet 4.5.
+    """
+
+    def __init__(
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)